src/linkify/utils/options.js

```diff
@@ -23,13 +23,13 @@ function Options(opts) {
 	this.formatHref = opts.formatHref || defaults.formatHref;
 	this.nl2br = opts.nl2br || defaults.nl2br;
 	this.tagName = opts.tagName || defaults.tagName;
-	this.target = opts.target || defaults.target;
+	this.target = opts.target !== undefined ? opts.target : defaults.target;
```

Comment:
What do you think about using [`hasOwnProperty`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty) instead of the `!== undefined` check?

e.g., 

```js
this.target = opts.hasOwnProperty('target') ? opts.target : defaults.target;
```